src/mame/heathkit/h89.cpp

```diff
@@ -297,14 +297,14 @@ static INPUT_PORTS_START( h89 )
 	PORT_DIPSETTING( 0x10, "Disk I/O #1" )
 	PORT_DIPNAME( 0x20, 0x20, "Perform memory test at start" )  PORT_DIPLOCATION("SW501:6")
 	PORT_DIPSETTING( 0x20, DEF_STR( Off ) )
-	PORT_DIPSETTING( 0x00, DEF_STR( On ) )
+	PORT_DIPSETTING( 0x00, "On" )
```

Comment:
If you’re using `DEF_STR` for one of the on/off pair, you should probably use it for both.